README.md

```diff
@@ -59,33 +60,106 @@ Table of Contents
 
 Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
 
-## Setup:
-The following `$SHELL` variables should be defined:
+## Setup
 
-- `$BUILD_HOME`
-- `$DATA_PATH`
-- `$ELOG_PATH`
+The `$SHELL` variable `$ELOG_PATH` should be defined:
+
+```
+export ELOG_PATH=/your/favorite/elog/path
+```
+
+Remove and download the setup script to ensure you have the most up-to-date version:
+
+```
+rm -f setup_gemdaq.sh
+wget https://raw.githubusercontent.com/cms-gem-daq-project/sw_utils/master/scripts/setup_gemdaq.sh
+```
 
 Then execute:
 
 ```
-source $BUILD_HOME/cmsgemos/setup/paths.sh
-source $BUILD_HOME/gem-plotting-tools/setup/paths.sh
+source setup.sh -c <cmsgemos tag> -g <gem-plotting tag> -G <gem-plotting dev version optional> -v <vfatqc tag> -V <vfatqc dev version optional>
 ```
 
-If this is the first time you are executing the above commands on `lxplus` it will create a `python v2.7` `virtualenv` for you.  It may take some time to download the necessary packages so be patient and do not interrupt the installation.  To disable this python env execute:
+If a development version is not to be used (normal case), you can drop the options with capital letters (`-G`, `-V`). If this is the first time you are executing the above command, it will create a Python `virtualenv` for you and install the `cmsgemos`, `gemplotting` and `vfatqc` packages. It may take some time to download them, so be patient and do not interrupt the installation.
+
+> **Example**
+>
+> ```
+> source setup_gemdaq.sh -c 0.3.1 -g 1.0.0 -G 5 -v 2.0.0 -V 3
+> ```
+>
+> This command will install the following packages:
+>
+> * `cmsgemos` version 0.3.1 (`-c 0.3.1`)
+> * `gemplotting` version 1.0.0-dev5 (`-g 1.0.0 -G 5`)
+> * `vfatqc` version 2.0.0-dev3 (`-v 2.0.0 -V 3`)
```

Comment:
Remove leading `>` as this will be rendered by MarkDown as quote mode (not applicable here) rather than preformatted mode (desired).

Reply:
Yes, I checked it on your GitHub page and removed the comment as can be seen.